cluster/src/main/java/org/apache/iotdb/cluster/log/applier/BaseApplier.java

```diff
@@ -102,6 +104,16 @@ private void handleBatchProcessException(BatchProcessException e, PhysicalPlan p
           && plan instanceof BatchPlan) {
         ((BatchPlan) plan).setIsExecuted(i);
       }
+
+      if (plan instanceof DeleteTimeSeriesPlan) {
+        if (status != null && status.getCode() != TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
+          if (status.getCode() != TSStatusCode.TIMESERIES_NOT_EXIST.getStatusCode()) {
+            logger.info("{} doesn't exist, it may has been deleted.", plan.getPaths().get(i));
+          } else {
+            needThrow = true;
```

Comment:
As delete operation doesn't need to be performd once again, maybe we can directly throw exception here ?

Reply:
1. First mistake has been modified.(status.getCode() == TSStatusCode.TIMESERIES_NOT_EXIST.getStatusCode())
2.Delete operation won't be performed once again, but if exeception is thrown directly here, batch exception won't be catched correctly in coordinator.
![image](https://user-images.githubusercontent.com/82880298/125305384-e26b9900-e360-11eb-9356-59e80adbb2f4.png)

Reply:
Sorry I don't get your meaning.
It seems once we set `needThrow` to `true` here, It will always be thrown in the last?  below codes are irrelevant with  delete operations, so I still don't know why we can not throw this exception here or before below codes?
```
boolean needRetry = false;
    for (int i = 0, failingStatusLength = failingStatus.length; i < failingStatusLength; i++) {
      TSStatus status = failingStatus[i];
      if (status != null
          && (status.getCode() == TSStatusCode.STORAGE_GROUP_NOT_EXIST.getStatusCode()
              || status.getCode() == TSStatusCode.UNDEFINED_TEMPLATE.getStatusCode())
          && plan instanceof BatchPlan) {
        ((BatchPlan) plan).unsetIsExecuted(i);
        needRetry = true;
      }
    }
    if (needRetry) {
      executeAfterSync(plan);
      return;
    }
```

Reply:
Delete operations indeed succeed, but the client may receives a error message which may confuse user. 


![image](https://user-images.githubusercontent.com/82880298/126255831-ef30058b-b2af-4aa1-a7f4-f37dee938fef.png)
2. 506

Reply:
OK, I see your meaning : DeleteOpearion can not throw exception here.